### DISCUSSION TITLE
What Is the Extra Region Under Tiddler Body

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#8429] [Q&A] [answered]
* Open 
* Create a new tiddler, put a line of text and save
* Right click on the new tiddler and select Inspect (I use Edge)
* See DevTool page

![image](https://github.com/user-attachments/assets/b6ddc967-8ceb-4faa-96ce-b320d5c9d7d4)

What is 

```
<ul class="multi-columns"></ul>
```

after

```
<div class="tc-tiddler-body tc-reveal"><p>Hello</p><p>I am a test!</p></div>
```

**Comment by pmario** [comment]
I think it's an artefact of [$:/editions/tw5.com/empty-tag-node-template](https://tiddlywiki.com/#%24%3A%2Feditions%2Ftw5.com%2Fempty-tag-node-template) Which  automatically shows a list of all tiddlers that are tagged with the currentTiddler.

IMO it should be fixed. If there is nothing to show, we should hide all elements.

**Reply by pmario** [reply, marked as answer]
IMO it needs to look like this

```
<$list filter='[<storyTiddler>!has[text]!has[tags]] :filter[tagging[]]'>
The following tiddlers are tagged with <<tag>>:

<<list-links filter:"[<storyTiddler>!has[text]!has[tags]tagging[]]" class:"multi-columns">>
</$list>
```

and instead of the list it should use `<% if [<storyTiddler>!has[text]!has[tags]tagging[]] %>`

**Reply by kookma** [reply]
Yes! They should be not there if not used!
I see this is an issue in tw5.con and NOT in empty.html

Thank you

**Comment by kookma** [comment]
Fixed in #8432
